src/agent/agent_instance.hpp

```diff
@@ -89,6 +89,14 @@ class AgentInstance
      */
     void Process(const otSysMainloopContext &aMainloop);
 
+    /**
+     * This method return mNcp pointer.
+     *
+     * @retval  the pointer of mNcp.
+     *
+     */
+    Ncp::Controller *getNcp() { return mNcp; }
```

Comment:
style: All void functions or methods shall explicitly declare and specify the void type keyword.